understanding/focus-not-obscured-minimum.html

```diff
@@ -56,116 +56,76 @@
+            <h1><span class="standalone-resource__type-of-guidance">解説書 達成基準 2.4.11:</span>隠されないフォーカス (最低限) (レベル AA)
             </h1>
             <aside class="box">
-               <header class="box-h  box-h-icon">Success Criterion (SC) </header>
+               <header class="box-h  box-h-icon">達成基準</header>
                <div class="box-i">
-                  <p>When a <a href="#dfn-user-interface-component">user interface component</a> receives keyboard focus, the component is not entirely hidden due to author-created
-                     content.
+                  <p><a href="#dfn-user-interface-component">ユーザインタフェース コンポーネント</a>がキーボードフォーカスを受け取るとき、コンテンツ制作者が作成したコンテンツによって、そのコンポーネントの全体が隠されるようなことがない。
                   </p>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>Where content in a configurable interface can be repositioned by the user, then only
-                        the initial positions of user-movable content are considered for testing and conformance
-                        of this Success Criterion.
+                     <p class="note-title marker">注記</p>
+                     <p>設定可能なインタフェースの中にあるコンテンツが利用者によって再配置可能な場合、この達成基準のテスト及び適合性の対象として検討されるのは、利用者によって移動可能なコンテンツの初期位置だけである。
                      </p>
                   </div>
                   <div class="note">
-                     <p class="note-title marker">Note</p>
-                     <p>Content opened by the <em>user</em> may obscure the component receiving focus. If the user can reveal the focused component
-                        without advancing the keyboard focus, the component with focus is not considered visually
-                        hidden due to author-created content.
+                     <p class="note-title marker">注記</p>
+                     <p><em>利用者</em>が開いたコンテンツによって、フォーカスを受け取るコンポーネントが隠される場合がある。利用者がキーボードフォーカスを移動せずにフォーカスを持つコンポーネントを表示できる場合、そのフォーカスを持つコンポーネントは、コンテンツ制作者が作成したコンテンツによって隠されたとはみなされない。
                      </p>
                   </div>
                </div>
             </aside>
             <div class="excol-all"></div>
             <section id="brief">
-               <h2>In Brief</h2>
+               <h2>簡潔に</h2>
                <dl>
                   
-                  <dt>Goal</dt>
-                  <dd>Keep the focused item visible.</dd>
+                  <dt>目標</dt>
+                  <dd>フォーカスのある項目を見えるようにしておく。</dd>
                   
-                  <dt>What to do</dt>
-                  <dd>Ensure when an item gets keyboard focus, it is at least partially visible.</dd>
+                  <dt>何をするか</dt>
+                  <dd>項目がキーボードフォーカスを取得したときに、その項目が少なくとも部分的に見えることを保証する。</dd>
                   
-                  <dt>Why it's important</dt>
-                  <dd>People who can't use a mouse need to see what has keyboard focus.</dd>
+                  <dt>なぜ重要か</dt>
+                  <dd>マウスを使用できない人々は、キーボードにフォーカスがあるものを確認する必要がある。</dd>
```

Comment:
「キーボードにフォーカス」という表現は他の「keyboard focus」と揃えて「キーボードフォーカス」でよいと思いますが、それに合わせて、少し表現を変えてみましあ。いかがでしょうか。

```suggestion
                  <dd>マウスを使用できない人々は、キーボードフォーカスがどこにあるかを確認する必要がある。</dd>
```